scripts/get_das_info.py

```diff
@@ -207,6 +224,58 @@ def get_das_info(
             info_of_interest["nfiles"] = dataset_info.get("nfiles", "")
             info_of_interest["nevents"] = dataset_info.get("nevents", "")
 
+    if add_file_info:
+        file_infos = load_das_info(dataset, add_file_info=True)
+
+        empty_files = [
+            info["file"][0]["name"]
+            for info in filter(lambda info: info["file"][0]["nevents"] == 0, file_infos)
```

Comment:
This is probably an overly-paranoid comment, but in principle this might open up a possibility for inconsistency if a broken file is also empty. It might be safer to explicitely require a valid file here as well

```suggestion
            for info in filter(lambda info: info["file"][0]["is_file_valid"] == 0 and info["file"][0]["nevents"] == 0, file_infos)
```